specs/PermissionManagement.md

```diff
@@ -18,7 +18,8 @@ events for some permission kinds.
 The new APIs, `SetPermission` and `GetNonDefaultPermissionSettings`, provide
 all the information necessary to build a permission management page where a
 user can view and modify existing site permissions. The new permission kinds we will
-support are: local font list, automatic downloads, media autoplay, and file editing.
+support are: local font list, automatic downloads, media autoplay, file editing, and
+system exclusive MIDI access.
```

Comment:
From my reading, system exclusive MIDI messages are not the same thing as system exclusive MIDI access.

System exclusive MIDI access: "I can access MIDI and nobody else can."
System exclusive MIDI message access: "I can send and receive so-called 'system exclusive' messages."

System exclusive messages are OEM-defined messages that are typically used for things like updating firmware or installing patches. They require special permission because they can alter the behavior of the hardware, and received sysex messages can be used to fingerprint the system.

They are called "system exclusive" messages because each message is "exclusive" to the OEM that defines it. It does not appear to have anything to with sharing/exclusivity.

Reply:
Much of the MDN documentation is not explicit about this. The doc for [Navigator.requestMIDIAccess](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/requestMIDIAccess) is slightly clearer though and looks to agree with Raymond:

> sysex
> A [Boolean](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean) value that, if set to true, allows the ability to send and receive system exclusive (sysex) messages. The default value is false.

We should update docs and the API name throughout

Reply:
(Because if it's _system exclusive access_ then the name suggests it would need elevated/administrative powers to perform...)

Reply:
I haven't seen anything about special permissions reading about this feature so far. It seemed more like this name (sysex / system exclusive) needs to be interpreted in the context of the world of MIDI where these words have a specific meaning, but @vickiez should say if this is the case. Are any special permissions required to use this from a medium IL app? Or from an app in an appcontainer?

Reply:
There aren't special permissions required from what I can tell and I agree with the above that the term is describing the messages